app/subscriber/src/features/my-reports/admin/ReportAdmin.tsx

```diff
@@ -129,7 +129,7 @@ export const ReportAdmin: React.FC<IReportAdminProps> = ({ path: defaultPath = '
       if (existingReport) {
         setReport(toForm(existingReport));
       } else {
-        getReport(reportId, true)
+        getReport(reportId)
```

Comment:
I reversed this back to how it was before as we don't need report content in ReportAdmin page